frontend/src/metabase/query_builder/components/NativeQueryEditor.jsx

```diff
@@ -442,6 +451,7 @@ class NativeQueryEditor extends Component {
                 readOnly={readOnly}
                 setDatabaseId={this.setDatabaseId}
                 setTableId={this.setTableId}
+                requireWriteback={requireWriteback}
```

Comment:
this is a nasty bit of prop drilling so that we can disable certain database options only in the action creator, but not the question creator.

I wanted to show them as disabled instead of simply filtering them out to avoid users say "WTF, where are my databases?"